usr.sbin/newsyslog/newsyslog.conf.5

```diff
@@ -117,7 +117,9 @@ This does not consider the current log file.
 .It Ar size
 When the size of the log file reaches
 .Ar size
-in kilobytes, the log file will be trimmed as described above.
+in Kilobyte, or by using any suffix (e.g., K, M, G) supported by
```

Comment:
The word should be kilobytes (plural and with a lowercase "k"), the same for the prefix (not suffix).

Reply:
Sorry for that, I forgot to push.